src/syscalls/linux/x86_64.rs

```diff
@@ -329,7 +329,7 @@ syscall!(231, sys_exit_group, int_t);
 syscall!(232, sys_epoll_wait, int_t, *mut epoll_event, int_t, int_t);
 syscall!(233, sys_epoll_ctl, int_t, int_t, int_t, *mut epoll_event);
 syscall!(234, sys_tgkill, pid_t, pid_t, int_t);
-syscall!(235, sys_utimes, *mut char_t, *mut timeval);
+syscall!(235, sys_utimes, *const char_t, *mut timeval); // WARNING *mut char_t
```

Comment:
The declaration for the syscall stub is `char*`, but in both the Linux and OS X kernel sources, it's treated as a `const char*`. Moreover the signature for utime is `int utime(const char *filename, const struct utimbuf *times);`. So there's idiosyncrasy between the two interfaces.

I chose this over an unsafe cast of the `posix::utime` argument to a `*mut char_t`. Thoughts?